Sat Jun 17 15:25:56 CEST 2006  VMiklos <vmiklos@frugalware.org>
  * libfwgrubconfifg: typo fix in get_mbr_dev()
diff -rN -u old-utils/libfwgrubconfig/libfwgrubconfig.c new-utils/libfwgrubconfig/libfwgrubconfig.c
--- old-utils/libfwgrubconfig/libfwgrubconfig.c	2006-06-17 15:47:54.000000000 +0200
+++ new-utils/libfwgrubconfig/libfwgrubconfig.c	2006-06-17 15:47:54.000000000 +0200
@@ -226,7 +226,7 @@
 	if(!fp)
 		return(NULL);
 	while(fgets(line, PATH_MAX, fp))
-		if(++i>3)
+		if(++i>2)
 			break;
 	fclose(fp);
 	ptr=line;

